### DISCUSSION TITLE
Is it possible to rotate a window 90 degree ?

### DISCUSSION THREAD

**Discussion by abs0023** [i3/i3#5380] [Q&A] [answered]
I'm playing with an xfce-i3 setup on postmarket os, I'm looking for a work around to get a sort of "landscape mode" for devices that doesn't have the accelerometre working yet.( rotating the screen is possible via the setting but it doesn't rotate the touch screen too, it is possible to rotate the touch screen but it involves restarting the device wich is not a quick rotation process)

**Comment by Airblader** [comment, marked as answer]
I don't entirely follow the question, but no, you cannot rotate individual windows. You can use xrandr to rotate the entire screen though.

**Reply by abs0023** [reply]
Sorry my question was messy, but thank you for your answer I will look at xrandr.
